bom/pom.xml

```
@@ -4,55 +4,55 @@
   <modelVersion>4.0.0</modelVersion>
 
   <parent>
-    <groupId>org.openhab.addons.bundles</groupId>
+    <groupId>org.openhab.addons.bom</groupId>
     <artifactId>org.openhab.addons.zigbee.reactor</artifactId>
     <version>2.5.0-SNAPSHOT</version>
   </parent>
 
-  <artifactId>org.openhab.binding.zigbee.bom</artifactId>
+  <artifactId>org.openhab.addons.bom.zigbee</artifactId>
   <packaging>pom</packaging>
 
   <name>openHAB ZigBee :: BOM :: openHAB ZigBee Binding</name>
 
   <dependencyManagement>
     <dependencies>
       <dependency>
-        <groupId>org.openhab.addons.bundles</groupId>
+        <groupId>org.openhab.addons.bom</groupId>
         <artifactId>org.openhab.binding.zigbee</artifactId>
         <version>${project.version}</version>
       </dependency>
       <dependency>
-        <groupId>org.openhab.addons.bundles</groupId>
+        <groupId>org.openhab.addons.bom</groupId>
```

The dependency groupIds shouldn't be changed. The BOM should list the bundles that this repo provides.

```suggestion
        <groupId>org.openhab.addons.bundles</groupId>
```